express-notes-api/index.js

```diff
@@ -0,0 +1,165 @@
+const express = require('express');
+const fs = require('fs');
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  const data = require('./data.json');
+  const arr = [];
+
+  for (const entry in data.notes) {
+    arr.push(data.notes[entry]);
+  };
+  res.status(200).send(arr);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const data = require('./data.json');
+  const id = req.params.id;
+  let found = false;
+
+  if (id >= 1) {
```

Comment:
you need to be checking here if the id is an integer, and if it is greater than or equal to 1.